cogs5e/initiative/cog.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, bot):
         self.bot = bot
         self.nlp = NLPRecorder(bot)
         self.buttons = ButtonHandler(bot)
+        self.member_converter = commands.MemberConverter()
```

Comment:
We probably don't need to save a converter instance here - we can just create it when needed (see -L746)